install/kubernetes/microcks/templates/deployment.yaml

```diff
@@ -160,6 +160,11 @@ spec:
             mountPath: "/deployments/config/kafka/keystore"
             {{- end }}
           {{- end }}
+          {{- if .Values.microcks.customKeystore.enabled }}
+          - name: "{{ .Values.microcks.customKeystore.secretRef.secret }}"
+            mountPath: "/etc/java/java-17-openjdk/java-17-openjdk-17.0.9.0.9-2.el9.x86_64/lib/security/{{ .Values.microcks.customKeystore.secretRef.key }}"
```

Comment:
Can we change to something less tied to the imnage internals? Something more spotted as custom would make more sense IMO. Something like `/deployments/config/custom/secret` for example.

Reply:
Thanks for the new commit @romain66 !